geoportal/geoportailv3_geoportal/static-ngeo/js/backgroundlayer/backgroundlayer.html

```diff
@@ -11,3 +11,14 @@
     </ul>
   </div>
 </div>
+
+<div class="mvt-style-loader" ng-if="ctrl.activeMvt">
+  <label for="uploadMvtStyle" class="btn">Changer style vectorielle</label>
+  <input
+    class="btn btn-default"
+    type="file"
+    name="uploadMvtStyle"
+    id="uploadMvtStyle"
+    custom-on-change="ctrl.setCustomStyle">
+  <a class="btn btn-default" id="clearMvtStyle" ng-click="ctrl.clearCustomStyle()" ng-if="ctrl.hasCustomStyle()">Supprimer</a>
+</div>
```

Comment:
These texts should be translatable, right?

Reply:
Yes, I did the change.